skins/cat17/src/sass/layouts/pages/use_of_funds.scss

```diff
@@ -0,0 +1,59 @@
+.page-use-of-funds {
+	.inline {
+		display: flex;
+		flex-direction: row;
+		flex-wrap: wrap;
+		justify-content: space-between;
+		align-items: center;
+	}
+	i {
+		float: right;
+		font-size: 24px;
+	}
+	.section-title {
+		font-size: 16px;
+		margin-bottom: 1em;
+		margin-top: 2em;
+	}
+	.subtitle {
+		font-size: 12px;
+		margin-bottom: 1em;
+		margin-top: 1em;
+	}
+	.money-progress {
+		border: 1px solid #00adee;
+		background: #00adee;
+		width: 50%;
+		height: 50px;
+		padding: 1em;
+		font-size: 16px;
+	}
+	.border {
+		border: 1px solid #00adee;
+		border-top: none;
+		width: 100%;
+		padding: 1em;
+	}
+	.border-bottom {
+		border-bottom: 1px solid #00adee;
+	}
+	.description {
+		font-size: 12px;
+	}
+	.list-menu {
+		li {
+			padding-right: 0;
+		}
+		@media( max-width: 677px ) {
+			margin-top: 50px;
```

Comment:
On Safari, this margin is too small. I can't figure out why, tough, maybe because the image is missing.

Reply:
If the rest of the spacing looks ok, then this should be in `em` instead of `px`. Hopefully that will fix the issue. I've set it to `3m` which is approximately `50px`.